.travis.yml

```diff
@@ -1,6 +1,9 @@
 sudo: required
 dist: trusty
 
+notifications:
+  email: false
```

Comment:
I would leave it on for now. I really wish Travis supported turning it off from the settings on the website. I'll remove this for now.